internal/cmd/api-key/command.go

```diff
@@ -89,9 +89,13 @@ func (c *command) getAllUsers() ([]*ccloudv1.User, error) {
 	if auditLog := v1.GetAuditLog(c.Context.Context); auditLog != nil {
 		serviceAccount, err := c.Client.User.GetServiceAccount(context.Background(), auditLog.GetServiceAccountId())
 		if err != nil {
-			return nil, err
+			// ignore 403s so we can still get other users
+			if !strings.Contains(err.Error(), "Forbidden Access") {
+				return nil, err
+			}
```

Comment:
Would it be more correct to check that the user's role is not "Developer" before calling the API?

Reply:
Or maybe we can make a whitelist of roles which are able to access the audit log service account?

Reply:
+1

Reply:
I think maintaining a list of roles that can or cannot access the audit log service account sounds like a lot of toil - would that mean every time we add a new role that can describe service accounts we would need to update this list? I feel like the best way to check if a user has permission to describe a service account would just be to try and call it, and skip over it if it doesn't have permission.